frontend/src/components/MeetingRoom/ToolbarOverflowMenu/ToolbarOverflowMenu.tsx

```diff
@@ -47,6 +47,14 @@ const ToolbarOverflowMenu = ({
   const participantCount =
     subscriberWrappers.filter(({ isScreenshare }) => !isScreenshare).length + 1;
   const isReportIssueEnabled = import.meta.env.VITE_ENABLE_REPORT_ISSUE === 'true';
+
+  const handleCloseMenu = (onClick?: () => void) => () => {
+    if (onClick) {
+      onClick();
+    }
+    handleClickAway();
+  };
+
   return (
     <Popper
       open={isToolbarOverflowMenuOpen}
```

Comment:
feel free to clean up some props while you're here. This is the name of the component, the prop can just be `isOpen`